packages/terra-application-navigation/src/tabs/_TabRollup.jsx

```diff
@@ -1,3 +1,4 @@
+/* eslint-disable jsx-a11y/role-supports-aria-props */
```

Comment:
This seems shady.

Reply:
I added it as the `TabRollup` return includes `area-haspopup`, which throws a linting error as it isn't supported by the role link according to `eslintjsx-a11y/role-supports-aria-props`. I didn't want to remove the line because I didn't want to cause unintended consequences for `TabRollup`.

Reply:
You'll want to look into as least switching the role to "button", but i think the proper fix could be some consideration also the "menubutton" role as well, and that will take corresponding changes to the PopupMenu and PopupMenuListItem. A guide can be found here:
https://www.w3.org/TR/wai-aria-practices/examples/menu-button/menu-button-links.html

Reply:
How about a compromise, log an issue to fix this and leave the ignore for now to get the repo building. 

Post back here with the issue please.